README.rst

```diff
@@ -3,34 +3,51 @@ Fledge South Person detection
 ******************************
 
 This directory contains a South service plugin that a detects person in the live video stream 
-and generates reading object with type of object (in this case it is  person), bounding box cordinates
-and confidence score.
+and generates reading object with type of object (in this case it is  person), bounding box cordinates and confidence score.
 The plugins also draws the bounding box on the detection window, and also displays confidence scores, 
 FPS (Frames Per Second) & type of object detected (Person). 
-In the reading the bounding box consists of {x,y} for top-left corner and bottom right corner of this rectangle. These coordinates values are based on a 640x480 canvas.
+In the reading the bounding box consists of {x,y} for top-left corner and bottom right corner of this rectangle. These coordinates
+values are based on a 640x480 canvas.
 
---------------------
-Plugin config params
---------------------
+-------------------------------
+Plugin Configuration Parameters
+-------------------------------
 
-TFlite model file:
+TFlite Model File:
         This is the name of the tflite model file that should be placed in
-        python/fledge/plugins/south/person_detection/model directory. Its default value is detect.tflite , The file name will different if a Coral  Edge TPU is connected (detect_edgetpu.tflite).
+        python/fledge/plugins/south/person_detection/model directory. Its default value is detect_edgetpu.tflite.
+        If a Coral Edge TPU is not being used, the file name will be different (i.e. detect.tflite).
 
-Labels file:
+Labels File:
         This is the name of the labels file that was used when training the above
         model, this file should also be placed in same directory as the model.
 
 Asset Name:
         The name of the asset used for the readings generated by this plugin.
 
-Enable Edge TPU :
-        If you want to use Coral Edge TPU then enable this config parameter. Also
-        change the name of the model file to detect_edgetpu.tflite
+Enable Edge TPU:
+        If you don't want to use Coral Edge TPU then disbale this configuration parameter.
```

Comment:
*disable

Reply:
Language should be "Indicates whether to use edge TPU for inference".